_includes/getDateWithStartAndEndTimeFromEvent.html

```diff
@@ -0,0 +1 @@
+{% assign timeOfEventDate = include.event.date | date: "%H:%M:%S" %}{% assign dateWithStartTime = include.event.date %}{% assign durationInSeconds = include.event.duration | times : 3600 %}{% if timeOfEventDate == '00:00:00' %}{% assign dateWithStartTime = include.event.date | date : "%Y-%m-%d " | append : include.event.startTime | date: '%s' %}{% capture dateWithStartTime %}{% include addOffsetToUTC.html time=dateWithStartTime offset=1 %}{% endcapture %}{% endif %}{% assign = dateWithEndTime = dateWithStartTime | date: "%s" | plus : durationInSeconds %}
```

Comment:
Da stecken JUG Darmstadt spezifische Inhalte drin. Wir haben meines Wissens keine duration. Aktuell gehen die Termine Kalender von 20:00 bis 20:00 Uhr

Reply:
Behoben in 5b9f73fe7d8b23bdbf01f7b9020bfdce51a309c9